.gitignore

```diff
@@ -33,4 +33,6 @@ pom.xml.versionsBackup
 .settings/
 
 **/__pycache__/*
-venv/
\ No newline at end of file
+venv/
+*.gradle
```

Comment:
Why are we ignoring gradle file here? this project is maven based, you wont have any gradle files in the directory.